OCP-Infrastructure/helm/hogajama/templates/hogajama-admin-svc.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: hogajama-admin
+  labels:
+    app: hogajama
+    app.kubernetes.io/component: hogajama
+    app.kubernetes.io/instance: hogajama-admin
```

Comment:
wenn die Installation über helm erfolgt, würde ich die Labels in eine Helper Function auslagern. Ist gängiges best-practice. 
Genauso in zeile 16 die Selector-Labels

Reply:
Darüber kann man diskutieren, auch wo Helm sinnvoll ist und wo nicht (bei Kafka ist kein Helm). Nachdem das vorher schon überall so gemacht wurde und man die Welt nicht in einem Pull Request retten kann, möchte ich da nichts ändern.